spec/requests/generic_object_definitions_spec.rb

```diff
@@ -414,7 +408,7 @@
 
       expected = {
         'results' => a_collection_including(
-          a_hash_including('id' => object_def.id.to_s, 'properties' => a_hash_including('attributes' => {})),
+          a_hash_including('id' => object_def.id.to_s, 'properties' => a_hash_including('attributes' => {'powered_on' => 'boolean'})),
```

Comment:
Similarly, this test had to be updated with the specifics of an object that was meant to be standing in for something ("an object that has attributes"), but any change to that trait would cause this to fail, even though could still serve perfectly well as "an object that has attributes". I think here and in any similar test it would be better to be decoupled from the trait.